socorro/external/pubsub/crashqueue.py

```diff
@@ -10,6 +10,11 @@
 from google.cloud import pubsub_v1
 
 
+# Maximum number of messages to pull from a Pub/Sub topic in a single pull
+# request
+PUBSUB_MAX_MESSAGES = 5
```

Comment:
Originally, this was 1. So every time, it does a synchronous blocking I/O pull request for a single message and gets either 0 or 1 messages and it does this a lot. Bleh.

The key issue here is we want standard items and priority items and reprocessing items to get processed evenly-ish. The tricky one is priority crash reports--we want them processed asap. So I think this changes it so that a priority crash report gets processed within a minute and that's probably fine since it doesn't happen often.